lib/experience-layout.js

```diff
@@ -13,12 +13,13 @@ const c = require('chalk');
 const VIEW_HEADERS = [ 'Page Name', 'Layout Name' ];
 
 module.exports = async (page, command) => {
+  const opts = command.opts ? command.opts() : {};
   const { apiToken, applicationId, api, appUrl } = await loadConfig();
   if (!applicationId || !apiToken) { return; } // config did not lock or did not load correctly.
   const pageQuery = { applicationId, viewType: 'page' };
-  if (page || command.pattern) {
+  if (page || opts.list) {
```

Comment:
why the change from pattern -> list?

Reply:
so the option is `.option('-l, --list <pattern>', 'pages that match this pattern will be listed with their layout')` I dont think this ever worked? or it got changed in the upgrade. but either way it should be list not pattern.